app/assets/stylesheets/components/_navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--yellow);
+  position: fixed;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: 100vw;
+  height: 100px;
+}
+
+.logo {
+  font-family: eds-market-bold-slant, sans-serif;
+  font-size: 16pt;
+  line-height: 17pt;
+  color: var(--blue);
+  text-align: center;
+}
+
+.design-elements {
+  font-family: Eds Market Design Elements;
+}
+
+.nav-right {
+  display: flex;
```

Comment:
**thought**: I feel like we're abusing `flex`. Are there places where we could get away with `block` or any default layout model?

Reply:
In this case I do think it serves a purpose, though it's used in two different ways. What would be considered abusing it, and why is that a problem?

<img width="868" alt="Screen Shot 2022-08-29 at 16 20 52" src="https://user-images.githubusercontent.com/87862340/187281454-170ed572-bb20-4e7f-8332-ce777533b2db.png">
<img width="1120" alt="Screen Shot 2022-08-29 at 16 21 10" src="https://user-images.githubusercontent.com/87862340/187281464-da0aa0f5-6489-4996-8d9b-8cbebc8c1117.png">

Reply:
Thinking back it's the old man in me speaking. I tend to prefer not adding styles to things when possible, so I usually think in this sequence:

1. Can I rely on the normalized behavior from `_base.css`?
2. Can I rely on the elements inheritance?
3. What's the minimum amount of rules to achieve what I'm trying to do here?

In this specific case you could:
- Mobile: Make children of the element `display: block`
- Desktop: Make children of the element `display: inline-block`

And nothing else to guarantee the positioning you want.

All that said, there is performance penalties, but they should be minimal, so I'm fine if you prefer things the way the are, this is mostly a matter of taste.

Reply:
I see, since I tend to over rely on flexbox, I changed it to use `block` and `inline-block`.
I did have, however to add some extra changes to get it to work.

desktop:
I had to add a `display: inline` to the p, and a paddings to the first link.

mobile:
I had to add `display: block` to everyone, `text-align: right` and `padding: 0` to the links.